docs/guides/completions.md

```diff
@@ -15,12 +20,19 @@ response = Completion.create(
     max_new_tokens=10,
     temperature=0.2,
 )
+
+print(response.json())
+# '{"request_id": "c4bf0732-08e0-48a8-8b44-dfe8d4702fb0", "output": {"text": "________ and I am a ________", "num_completion_tokens": 10}}'
+
+print(response.output.text)
+# ________ and I am a ________
 ```
 
-The _model_name_ is the LLM to be used (see [Model Zoo](../../model_zoo)).
-The _prompt_ is the main input for the LLM to respond to.
-The _max_new_tokens_ parameter is the maximum number of tokens to generate in the chat completion.
-The _temperature_ is the sampling temperature to use. Higher values make the output more random, while lower values will make it more focussed and deterministic.
+**model:** The LLM you want to use (see [Model Zoo](../../model_zoo)).  
```

Comment:
Should we make these bullets?

Reply:
Which do you prefer?

<img width="780" alt="Screenshot 2023-07-17 at 11 01 10 PM" src="https://github.com/scaleapi/llm-engine/assets/7238671/ff71ee0b-99fa-406f-b5d7-aad911cf89dc">
<img width="804" alt="Screenshot 2023-07-17 at 11 01 31 PM" src="https://github.com/scaleapi/llm-engine/assets/7238671/03f575fb-ab44-4967-a576-dccdc47a22f7">

Reply:
I though the bullets added unnecessary spacing

Reply:
Oooh yeah you're right